### ISSUE TITLE
[Feature] Ammo chooser UI for the cargo bay

### ISSUE THREAD

**Issue by Ligraph** [pyfa-org/Pyfa#1210]
Not a priority feature by any means, but I've been creating and lot of fits lately and it would be nice if there was a better way to add ammo to the cargo bay.  Even if it was similar to the dropdown lists on the weapons.  

The two easiest ways I thought of was to have a dropdown in the cargo bay where you choose a weapon, then the ammo and it adds it to cargo, or to have an option on the weapons to add the ammo to the cargo bay.

**Comment by blitzmann** [comment]
Good idea in general, I'll take a look at what can be done about this :)
